connector_importer/components/importer_csv_std.py

```diff
@@ -0,0 +1,175 @@
+# Copyright 2019 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from odoo.addons.component.core import Component
+
+from ..log import logger
+
+
+class RecordImporterCSVStd(Component):
+    """CSV Standard importer for records.
+
+    This importer is used to import standard CSV files, using the `load()`
+    method of Odoo.
+    """
+
+    _name = 'importer.record.csv.std'
+    _inherit = ['importer.record']
+    _break_on_error = True  # We want the import to stop if an error occurs
+    _apply_on = None
+    _use_xmlid = True
```

Comment:
add a TODO here and an item in the roadmap: we can support such easy options in the configuration of the import type so that you don't have to define a customer importer just to say "use_xmlid".

For instance, settings could like like:

`product.template::template.importer.component.name::use_xmlid=true,opt2=0`

then the parser of the settings can transform it into a `importer_options` dict to be passed on init.